components/Common/CodeBox/__tests__/index.test.mjs

```diff
@@ -0,0 +1,102 @@
+import * as Toast from '@radix-ui/react-toast';
+import { render, screen, waitFor, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { IntlProvider } from 'react-intl';
+
+import CodeBox from '@/components/Common/CodeBox';
+
+jest.mock('@heroicons/react/24/outline', () => ({
```

Comment:
Is this mock needed... Why?

Reply:
Because assigning a `data-testid` to the icon allow us to ensure the icon is rendered on the DOM. Otherwise, it wouldn't be possible to differentiate more than one `svg` element.

Still, if you believe it is not needed I can just remove it, no problem. This is just a pattern I'm used to following.